test/sql/function/test_stringfunctions.cpp

```diff
@@ -131,3 +131,20 @@ TEST_CASE("UPPER/LOWER test", "[function]") {
 	result = con.Query("select LOWER(b) FROM strings");
 	REQUIRE(CHECK_COLUMN(result, 0, {"world", Value(), "rÄcks"}));
 }
+
+TEST_CASE("REVERSE test", "[function]") {
+	unique_ptr<QueryResult> result;
+	DuckDB db(nullptr);
+	Connection con(db);
+	con.EnableQueryVerification();
+
+	REQUIRE_NO_FAIL(con.Query("CREATE TABLE strings(a STRING, b STRING)"));
+	REQUIRE_NO_FAIL(con.Query("INSERT INTO strings VALUES ('Hello', 'World'), "
+	                          "('HuLlD', NULL), ('MotörHead','RÄcks')"));
+
+	result = con.Query("select REVERSE(a) FROM strings");
+	REQUIRE(CHECK_COLUMN(result, 0, {"olleH", "DlLuH", "daeHrötoM"}));
+
+	result = con.Query("select REVERSE(b) FROM strings");
+	REQUIRE(CHECK_COLUMN(result, 0, {"dlroW", Value(), "skcÄR"}));
```

Comment:
Perhaps also check with invalid UTF strings?

Reply:
Also missing the empty string as a test.

Reply:
Invalid UTF strings should not be inside the database, so they should not be checked here. Empty string is a good suggestion though!